<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MasterView</class>
 <widget class="QWidget" name="MasterView">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>551</width>
    <height>353</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MasterView</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QPushButton" name="btn_return">
       <property name="text">
        <string>返回</string>
       </property>
       <property name="icon">
        <iconset resource="imgs.qrc">
         <normaloff>:/images/text-wrap.png</normaloff>:/images/text-wrap.png</iconset>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Orientation::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QLabel" name="label">
       <property name="text">
        <string>TextLabel</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_2">
       <property name="orientation">
        <enum>Qt::Orientation::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="btn_Logout">
       <property name="text">
        <string>注销</string>
       </property>
       <property name="icon">
        <iconset resource="imgs.qrc">
         <normaloff>:/images/logout-box-line.png</normaloff>:/images/logout-box-line.png</iconset>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QStackedWidget" name="stackedWidget">
     <property name="styleSheet">
      <string notr="true"/>
     </property>
     <property name="currentIndex">
      <number>-1</number>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="imgs.qrc"/>
 </resources>
 <connections/>
</ui>
